apps/ui/src/components/Ui/Message.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+defineProps<{
+  type: 'info' | 'danger';
+  learnMoreLink?: string;
+}>();
+</script>
+
+<template>
+  <div
+    class="border rounded-md p-[20px] flex gap-1"
+    :class="{
+      'text-skin-danger border-skin-danger': type === 'danger'
+    }"
+  >
+    <IH-information-circle class="shrink-0" />
+    <div class="leading-5">
+      <slot />
+      <a
+        v-if="learnMoreLink"
+        :href="learnMoreLink"
+        target="_blank"
+        class="flex items-center text-skin-link mt-1 gap-1 w-fit"
+      >
+        Learn more
+        <IH-arrow-top-right-on-square />
```

Comment:
This is indeed from Figma, but on v2, we're now using the 45deg arrow without the square box for external link.

Better to follow the v2 format